### DISCUSSION TITLE
How to set a wide scroll bar, instead of thin?

### DISCUSSION THREAD

**Discussion by alexandrehtrb** [AvaloniaUI/Avalonia#15235] [Q&A]
Hello,

How do I configure a ScrollBar to always be wide, instead of thin?
Like in the image below, I want the vertical scroll bar to be wide, just like the horizontal scroll bar is.

![Screenshot_20240404_150623](https://github.com/AvaloniaUI/Avalonia/assets/27026741/af71eb98-d81d-479f-98a3-c035a50985d0)

**Comment by timunie** [comment]
I don't get your point? You can set `AllowAutoHide="false"` on the ScrollViewer if that was your question.

**Reply by alexandrehtrb** [reply]
I already set `AllowAutoHide="false"` (including in internal ScrollViewer, as mentioned [here](https://github.com/AvaloniaUI/AvaloniaEdit/issues/146#issuecomment-891383379)), but it didn't work.

I want a "fat" vertical scroll bar, like this:

![Pororoca_3d4v7xAHx7](https://github.com/AvaloniaUI/Avalonia/assets/27026741/7ec77b0f-ed31-45dd-b815-b3cf930a1e82)

The thing is, it appears that when there is horizontal and vertical scroll, the vertical scroll bar becomes thin and only the horizontal is fat:

![Pororoca_3psknBP11I](https://github.com/AvaloniaUI/Avalonia/assets/27026741/47122cd7-3135-435d-b725-cbe3186eb377)

I posted this question here instead of AvaloniaEdit because I am having this same issue with DataGrid

**Reply by timunie** [reply]
Please lookup the original styles. Maybe the controls you mention needs a different setter here.

**Reply by stevemonaco** [reply]
I have no issue with the following in Fluent Theme:

```xml
<Window.Styles>
    <Style Selector="ScrollBar">
        <Setter Property="AllowAutoHide" Value="False" />
    </Style>
</Window.Styles>

<Grid>
    <ScrollViewer HorizontalScrollBarVisibility="Visible" VerticalScrollBarVisibility="Visible">
        <TextBlock Text="{Binding Greeting}" />
    </ScrollViewer>
</Grid>
```
